src/base/QXmppRosterIq.cpp

```diff
@@ -253,6 +280,34 @@ void QXmppRosterIq::Item::setSubscriptionTypeFromStr(const QString& type)
         qWarning("QXmppRosterIq::Item::setTypeFromStr(): invalid type");
 }
 
+/// Returns whether this is a MIX channel.
+
+bool QXmppRosterIq::Item::isMixChannel() const
+{
+    return m_isMixChannel;
+}
+
+/// Sets whether this is a MIX channel.
+
+void QXmppRosterIq::Item::setIsMixChannel(bool isMixChannel)
+{
+    m_isMixChannel = isMixChannel;
+}
+
+/// Returns the participant id for this MIX channel.
+
+QString QXmppRosterIq::Item::participantId() const
```

Comment:
Should this be `mixParticipantId` to keep everything MIX-related with a consistent prefix?

Reply:
done